matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/identity/IdentityBulkLookupTask.kt

```diff
@@ -144,4 +139,11 @@ internal class DefaultIdentityBulkLookupTask @Inject constructor(
             )
         }
     }
+
+    private val sha256 by lazy { MessageDigest.getInstance("SHA-256") }
+
+    @OptIn(ExperimentalStdlibApi::class)
+    private fun String.toSha256(): String {
+        return sha256.digest(toByteArray()).toHexString()
```

Comment:
Found it surprising that we use toHexString here. The legacy sha was returning a base64 string no? [.getInstance("SHA-256")](https://gitlab.matrix.org/matrix-org/olm/-/blob/master/tests/test_olm_sha256.cpp#L16)

Reply:
Good catch thanks! f726d16ce0e59ba600aa5d1b0a3622b440aadc69

Reply:
Thx, maybe just a nit, it could be named `convertToSha256Base64Encoded`